README.md

```diff
@@ -109,8 +122,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
-
+fsm_DylanWeeks.jpg
```

Comment:
# 4

How you translated your grammar into this is beyond me :)

This is a correct FSM, but it has an unnecessary state: S2 could be removed and S1 could loop back to itself when it receives adjectives.